metro.config.js

```diff
@@ -1,4 +1,37 @@
 // Learn more https://docs.expo.io/guides/customizing-metro
 const { getDefaultConfig } = require('expo/metro-config');
 
-module.exports = getDefaultConfig(__dirname);
+module.exports = (async () => {
+  const {
+    resolver: { assetExts }
+  } = await getDefaultConfig(__dirname);
+
+  return {
+    transformer: {
+      getTransformOptions: async () => ({
+        transform: {
+          experimentalImportSupport: false,
+          inlineRequires: true
```

Comment:
what does this mean? can you explain or point to some documentation please.

Reply:
those two links do not say anything about the two values. how can they help me to understand? i think we need other links or ask the viro community about this?

Reply:
if you create a project with `npx react-native init` these lines come automatically. I don't think this has anything to do with viro, unfortunately I couldn't find a detailed documentation.

Reply:
i asked at metro repo for more details, let's see if someone answers.